lib/v2/gzh360/gzh.js

```diff
@@ -0,0 +1,60 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const { mpIdEncode, mpIdDecode, finishArticleItem } = require('./utils');
+const config = require('@/config').value;
+
+module.exports = async (ctx) => {
+    let name = ctx.params.name;
+    let id = name;
+
+    // in order to support RSSHub Radar, we need to accept both name and id
+    try {
+        name = mpIdDecode(name);
+    } catch {
+        id = mpIdEncode(name);
+    }
+
+    const rootUrl = 'http://web.gzh360.com';
+    const currentUrl = `${rootUrl}/gzh?id=${id}`;
```

Comment:
It seems we can grab more articles from `/gzh_articles`
```suggestion
    const currentUrl = `${rootUrl}/gzh_articles?id=${id}`;
```

Reply:
Thx. Since `/gzh_articles` and `/category` share the same webpage format, I've changed this route to grab them in almost the same way.